src/classBased/components/TodoItem.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+import styles from './TodoItem.module.css';
+
+/* eslint-disable */
+class TodoItem extends Component {
+  state = {
+    editing: false,
+  }
```

Comment:
Thank you for the suggestion. However, I was following the tutorial and that is how they did it. Again the class-based component is not what I am using. It's only there because the tutorial added it to the structure. Thank you